guide.qmd

```diff
@@ -268,8 +228,28 @@ guide_table(
 
 Life cycle management is simplified by reducing dependencies, and should comprise a central code repository.
 
+### Dependency Management
 1.  **Reduce dependencies** to simplify maintenance of your own package. Only depend on other packages that you trust and deem stable enough for the purpose, in order to avoid reinventing the wheel.
 
+
+2. **Use tools that track dependencies and their versions** as these are moving targets, i.e. they are periodically updated. Insufficient dependency management is a common reason for "it doesn't work on my computer" errors. 
+
+::: {.panel-tabset}
```

Comment:
can we split out the proposed layout change in another PR?

Reply:
What's your thoughts on the layout changes? Do you want to use the tabsets or keep as is? Once we align I can decide if it's worth the effort to split it out :)

Reply:
Ok. I'll remove the tabsets from this PR. 

Would we need the guide_tabset() if we replace it with tabsets throughout? I guess not? Unless there are some you want to keep?

Reply:
Idea would be to remove duplicate code - if using something like a new `guide_tabset()` can reduce code

Reply:
Not sure I follow - if we use inbuilt tabsets functionality I don't think we need any code, right?

Reply:
not necessarily, but we might save some manual copy/pasting of boiler plate tabsets code potentially, but we can see in other PR